web/src/components/pages/profile/avatar-setup/avatar-setup.css

```diff
@@ -127,22 +131,35 @@
 
     .fix {
         height: 240px;
-        width: 100%;
+        width: 508px;
     }
     .start-in {
         position: relative;
-        top: -8px;
+        top: -12px;
         width: 39px;
         height: 18px;
+        font-family: ;
```

Comment:
this looks a bit like zeplin boilerplate, do we need all these rules?

Reply:
yes, this is affecting the style if not mentioned !